relay/polkadot/src/weights/pallet_staking.rs

```diff
@@ -877,29 +850,38 @@ impl<T: frame_system::Config> pallet_staking::WeightInfo for WeightInfo<T> {
 	/// Storage: `Staking::ActiveEra` (r:1 w:0)
 	/// Proof: `Staking::ActiveEra` (`max_values`: Some(1), `max_size`: Some(13), added: 508, mode: `MaxEncodedLen`)
 	/// Storage: `Staking::Invulnerables` (r:1 w:0)
-	/// Proof: `Staking::Invulnerables` (`max_values`: Some(1), `max_size`: Some(641), added: 1136, mode: `MaxEncodedLen`)
+	/// Proof: `Staking::Invulnerables` (`max_values`: Some(1), `max_size`: None, mode: `Measured`)
 	/// Storage: `Staking::ErasStakersOverview` (r:1 w:0)
 	/// Proof: `Staking::ErasStakersOverview` (`max_values`: None, `max_size`: Some(92), added: 2567, mode: `MaxEncodedLen`)
-	/// Storage: `Session::DisabledValidators` (r:1 w:1)
+	/// Storage: `Staking::ErasStakersPaged` (r:1 w:0)
+	/// Proof: `Staking::ErasStakersPaged` (`max_values`: None, `max_size`: None, mode: `Measured`)
+	/// Storage: `Session::DisabledValidators` (r:1 w:0)
 	/// Proof: `Session::DisabledValidators` (`max_values`: Some(1), `max_size`: None, mode: `Measured`)
 	/// Storage: `Session::Validators` (r:1 w:0)
 	/// Proof: `Session::Validators` (`max_values`: Some(1), `max_size`: None, mode: `Measured`)
 	/// Storage: `Staking::ValidatorSlashInEra` (r:1 w:1)
 	/// Proof: `Staking::ValidatorSlashInEra` (`max_values`: None, `max_size`: Some(72), added: 2547, mode: `MaxEncodedLen`)
-	/// Storage: `Staking::OffenceQueue` (r:1 w:1)
-	/// Proof: `Staking::OffenceQueue` (`max_values`: None, `max_size`: Some(101), added: 2576, mode: `MaxEncodedLen`)
-	/// Storage: `Staking::OffenceQueueEras` (r:1 w:1)
-	/// Proof: `Staking::OffenceQueueEras` (`max_values`: Some(1), `max_size`: Some(2690), added: 3185, mode: `MaxEncodedLen`)
+	/// Storage: `Staking::SlashingSpans` (r:513 w:513)
+	/// Proof: `Staking::SlashingSpans` (`max_values`: None, `max_size`: None, mode: `Measured`)
+	/// Storage: `Staking::SpanSlash` (r:513 w:513)
+	/// Proof: `Staking::SpanSlash` (`max_values`: None, `max_size`: Some(76), added: 2551, mode: `MaxEncodedLen`)
+	/// Storage: `Staking::NominatorSlashInEra` (r:512 w:512)
+	/// Proof: `Staking::NominatorSlashInEra` (`max_values`: None, `max_size`: Some(68), added: 2543, mode: `MaxEncodedLen`)
+	/// Storage: `Staking::UnappliedSlashes` (r:1 w:1)
+	/// Proof: `Staking::UnappliedSlashes` (`max_values`: None, `max_size`: None, mode: `Measured`)
 	fn manual_slash() -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `514`
-		//  Estimated: `4175`
-		// Minimum execution time: 30_000_000 picoseconds.
-		Weight::from_parts(33_000_000, 4175)
-			.saturating_add(T::DbWeight::get().reads(10_u64))
-			.saturating_add(T::DbWeight::get().writes(4_u64))
+		//  Measured:  `20713`
+		//  Estimated: `1309653`
+		// Minimum execution time: 7_768_872_000 picoseconds.
+		Weight::from_parts(7_843_282_000, 0)
+			.saturating_add(Weight::from_parts(0, 1309653))
+			.saturating_add(T::DbWeight::get().reads(1549))
+			.saturating_add(T::DbWeight::get().writes(1540))
```

Comment:
1.5k reads and writes looks.. wrong.
it probably won't allow a single such extrinsic into a block